gym/envs/toy_text/kellycoinflip.py

```diff
@@ -93,7 +93,7 @@ def __init__(self, initialWealth=25, edgePriorAlpha=7, edgePriorBeta=3, maxWealt
         self.maxRounds = maxRounds
         self.rounds = self.maxRounds
         self.maxWealth = maxWealth
-        self._seed()
+        if reseed: self._seed()
```

Comment:
Checking, I've realized there's another way users could be confused/misused it: if you set the hyperparameters in init and then do resets, you'll get a new game with parameters drawn according to the default hyperparameters, but you probably expected the parameters to be drawn according to your original hyperparameter options. I don't know any better way to program this in Python but to store all the overriden options in the env object and just pass them into init on reset.